app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -216,6 +216,18 @@ protected void enterRootModule() {
         startActivityForResult(i, GET_COMMAND);
     }
 
+    private boolean useGridMenu(String menuId) {
+        if(menuId == null) {
+            menuId = org.commcare.suite.model.Menu.ROOT_MENU_ID;
+        }
+        if(DeveloperPreferences.isGridMenuEnabled()) { return true; }
+        String commonDisplayStyle = platform.getMenuDisplayStyle(menuId);
+        if("grid".equals(commonDisplayStyle)) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
replace conditional with
`return "grid".equals(commonDisplayStyle);`
then replace `"grid"` with a constant defined in the appropriate menu/style related class